app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -235,7 +235,7 @@ public function createOrder($reference, $session_quote_id = null)
             }
 
             if (!$is_shipping_set) {
-                $errorMessage = 'Shipping method not found';
+                $errorMessage = 'Shipping method not found. $service: ' . $service . ' $shipping_address: ' . var_export($shipping_address->debug(), true);
```

Comment:
let's put variables in metadata instead of dumping it into message body, so it's easier to read in bugsnag

Reply:
I'll change and re-push. Good call.